index.bs

```diff
@@ -345,6 +345,65 @@ and on when to use promises and when not to use promises,
 see <strong><a href="https://www.w3.org/2001/tag/doc/promises-guide">Writing
 Promise-Using Specifications</a></strong>.
 
+<h3 id="secure-context">Limit new and distinct APIs to secure contexts</h3>
+
+Many new APIs should be available only in
+<a href="https://w3c.github.io/webappsec-secure-contexts/">secure contexts</a>.
+This restriction exists for two reasons.
+First, it helps encourage Web content and applications
+to migrate to secure contexts.
+Second, it can restrict new APIs where authentication, integrity, or confidentiality
+are important to prevent substantial increases to the privacy or security risks of using the Web.
+For more detail, see the W3C TAG Finding on
+<a href="https://www.w3.org/2001/tag/doc/web-https">Securing the Web</a>.
+
+When the new feature is defined in
+<a href="https://heycam.github.io/webidl/">WebIDL</a>,
+specification authors can limit a feature to secure contexts
```

Comment:
I think this one actually meant can; this was a description of facts, not a conformance requirement.